### PR TITLE
Update kubeadm reference

### PR DESCRIPTION
**Pull request by tengqm**
Generator update PR: kubernetes-sigs/reference-docs#283, kubernetes-sigs/reference-docs#284
closes: #32790

### CODE REVIEW

File: content/en/docs/reference/config-api/kubeadm-config.v1beta3.md [hunk 1]
```diff
@@ -555,7 +555,7 @@ by kubeadm during <code>kubeadm join</code>.</p>
 <code>int32</code>
 </td>
 <td>
-   <p><code>bindPort</code> sets the secure port for the API Server to bind to.
+   <p><code>bindPorti</code> sets the secure port for the API Server to bind to.
```
Comment (sftim) [inline content/en/docs/reference/config-api/kubeadm-config.v1beta3.md:558]: `bindPorti`? Was https://github.com/kubernetes/website/commit/3fe7033047f520c1b801d5b6c405357bdf58ddf6 wrong?

Comment (tengqm) [reply content/en/docs/reference/config-api/kubeadm-config.v1beta3.md]: upstream source update proposed: kubernetes-sigs/reference-docs#284

**Comment by neolit123** [comment]
i don't see the bindPorti typo in here:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta3/types.go

/lgtm

**Comment by tengqm** [comment]
> i don't see the bindPorti typo in here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta3/types.go
> 
> /lgtm

It was introduced when I copy and edit the comment in tengqm/kubeconfig repo. I had to copy these type files out of k/k because the kubeadm config APIs are located inside cmd/kubeadm rather than a standalone package that can be imported.

**Pull request merged**